CMakeLists.txt

```diff
@@ -88,10 +88,25 @@ macro(jerry_add_compile_flags)
   jerry_add_flags(CMAKE_C_FLAGS ${ARGV})
 endmacro()
 
+# SHOULD THIS BE, SAY, AT THE TOP OF THE FILE?
+if(CMAKE_C_COMPILER_ID MATCHES "GNU")
```

Comment:
I was working off of the cmake documentation -- https://cmake.org/cmake/help/v3.0/variable/CMAKE_LANG_COMPILER_ID.html -- is there a better/conventional way to do this?